### ISSUE TITLE
CI

### ISSUE THREAD

**Issue by joshka** [Dark-Alex-17/managarr#2]
If you're after some easy CI config, take a look at https://github.com/joshka/github-workflows/
I use it for a bunch of crates

**Comment by Dark-Alex-17** [comment]
This is great! Thank you! 

I'm mostly caught up with work at the moment (you know, big projects with deadlines ☺ ), so that's the main holdup. But I can say that Radarr support is stable and essentially ready for alpha.

Since I'm giving an update, I may as well also mention the current efforts: I'm working to create a mock API for Transmission so that I can create a docker compose that encompasses an entire Servarr system. The mock API will make it possible to mimic downloads for E2E testing and demonstration purposes. 

So that is the main thing I'm working towards since it would be extremely useful to have a mock API for testing purposes when I want to try out things in managarr and ensure they work, especially with future developments and contributions.

**Comment by Dark-Alex-17** [comment]
I'm very happy to say that I've released Managarr into alpha. I used your workflows for the CI/CD, but copied them directly into the repo since I eventually want to extend the release to release to Homebrew, Chocolatey, and any other package managers that would be useful for users. I've credited you in the workflow definitions. 

Thank you for your suggestion. It made setting up the CI/CD FAR easier and quicker than I anticipated it being.
